noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/view/screen/SignUpScreen.kt

```diff
@@ -53,15 +55,16 @@ fun SignUpScreen(
     viewModel: RegisterViewModel
 ) {
 
-    val viewState = viewModel.authFlow.collectAsState(initial = null).value
-
-    when (viewState) {
+    when (val viewState = viewModel.authFlow.collectAsState(initial = null).value) {
         is ViewState.Loading -> LoaderDialog()
         is ViewState.Success -> {
             navController.navigate(
                 route = Screen.Notes.route,
                 builder = {
                     launchSingleTop = true
+                    popUpTo(Screen.SignUp.route) {
+                        inclusive = true
+                    }
                 }
             )
         }
```

Comment:
@carotkut94 I think this isn't working. I tested the built app and produced scenario as follows:

- Clean install application
- Launch app 
- Clicked on "Sign up"
- Entered credentials and clicked "Sign Up"
- (_Notes screen launched_) Press back 🔙 

After these procedures, the Sign Up screen was launched. This needs to be fixed.